mne/io/__init__.py

```diff
@@ -5,6 +5,7 @@
 #
 # License: BSD (3-clause)
 
+from .constants import FIFF
```

Comment:
why did you need this?

I prefer avoiding crowding too much the module levels (yes I know we already do it a lot...)